server/models/Conversation.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+const conversationSchema = new mongoose.Schema(
+  {
+    firstUser: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "user",
```

Comment:
```suggestion
      ref: "User",
```
Always use PascalCase for model names. However, I think changing it at this point may cause the app to break (so you can refrain from doing so) 😄

Reply:
Oh, I was just following the standard from the previous model. But if you insist I can use PascalCase from now onwards. 🙂